app/services/reporters/event_service_reporter.rb

```diff
@@ -73,12 +73,9 @@ def create_error_event(druid, version, process_name, moab_storage_root, error_me
       events_client_for(druid).create(
         type: 'preservation_audit_failure',
         data: {
-          host: Socket.gethostname,
-          invoked_by: 'preservation-catalog',
-          storage_root: moab_storage_root&.name,
-          actual_version: version,
-          check_name: process_name,
-          error: error_message
+          host: Socket.gethostname, invoked_by: 'preservation-catalog',
```

Comment:
I think this change here is less readable, because you have to parse out where the arguments are. Can this be reverted?